sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

```diff
@@ -394,6 +430,7 @@ The table below lists deprecated, that would soon be removed and their replaceme
 |Deprecated Descriptors | Replacement Descriptors |
 | ------------------------------- |--------------------------- |
 | info\:\:device\:\:ext\_intel\_device\_info\_uuid  | ext\:\:intel\:\:info\:\:device\:\:uuid |
+| info\:\:device\:\:ext\_intel\_pci\_device\_id     | ext\:\:intel\:\:info\:\:device\:\:pci\_device\_id |
```

Comment:
The `info::device::ext_intel_pci_device_id` descriptor never existed, so there is no point in deprecating it. You can just remove this line.

Reply:
Removed.